lib/components/AlbumScreen/item_info.dart

```diff
@@ -39,33 +33,32 @@ class ItemInfo extends StatelessWidget {
         //     ),
         //   ),
         // ),
-        if (item.type != "Playlist") ArtistChips(
-          baseItem: item,
-          useAlbumArtist: true,
-        ),
+        if (item.type != "Playlist")
+          ArtistChips(
+            baseItem: item,
+            useAlbumArtist: true,
+          ),
         IconAndText(
             iconData: Icons.music_note,
             textSpan: TextSpan(
-              text: AppLocalizations.of(context)!.songCount(itemSongs),
-              style: Theme.of(context).textTheme.bodyMedium,
```

Comment:
I found this out when trying to add download icons to the album list tiles that the RichText widget blocks text style transmission, but the Text.rich widget does not.  I changed that out in the IconAndText class and that fixed all these.

Reply:
Ahh, neat. So is there any reason to use `RichText` instead of `Text.rich()` in other places? Otherwise we can go ahead and replace them all and worry less about styling :D

Reply:
It appears the total style reset is the main point of RichText vs Text.rich, so I don't see why we would ever prefer it.

Reply:
Okay. But it can be done outside of this PR, in case there is some breaking UI change :)